syntax = "proto3";

package example.simple;

message Simple {
  uint32 id = 1;
  bool is_simple=2;
  string name = 4;
  repeated int32 sample_lists=3;
}